src/main/java/com/exadel/discount/service/CityService.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.discount.service;
+
+import com.exadel.discount.dto.CityDTO;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface CityService {
+    List<CityDTO> findAll();
```

Comment:
please add an empty line between class and method declaration

Reply:
Done